.github/workflows/ci.yml

```diff
@@ -176,8 +183,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "3.12"
+          - "3.13"
         os: [ ubuntu-latest, macos-latest, windows-latest ]
+        include:
+          - python-version: "3.7"
+            os: ubuntu-latest
+          - python-version: "3.7"
+            os: macos-13
```

Comment:
Python 3.7 isn't available for `macos-latest` (currently `macos-14`) anymore, so I extracted these.

Reply:
IMHO - you can remove python3.7 testing completely given that this is not supported anymore. Additionally, the `README` contains instructions on this lib versions that are still working with dead python versions.

Reply:
We are going to keep Python 3.7 support.